src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -16,8 +16,9 @@ public class SendPacketsAsync
         private readonly ITestOutputHelper _log;
 
         private IPAddress _serverAddress = IPAddress.IPv6Loopback;
-        // In the current directory
-        private const string TestFileName = "NCLTest.Socket.SendPacketsAsync.testpayload";
+        // Accessible directories for UWP app:
+        // C:\Users\<UserName>\AppData\Local\Packages\<ApplicationPackageName>\
+        private string TestFileName = Environment.GetEnvironmentVariable("LocalAppData") + @"\NCLTest.Socket.SendPacketsAsync.testpayload";
```

Comment:
Ah, Environment.SpecialFolder is not working right in Appcontainer right now. That's tracked.
But if you're just looking for a temp file name, this problem is already solved. You can use TempFile
https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/IO/TempFile.cs 
If it's convenient you can use a using statement to remove it afterwards.  It's not magic, it just uses temp path.